packages/flutter/lib/src/cupertino/switch.dart

```diff
@@ -136,6 +145,7 @@ class _CupertinoSwitchRenderObjectWidget extends LeafRenderObjectWidget {
       onChanged: onChanged,
       textDirection: Directionality.of(context),
       vsync: vsync,
+      dragStartBehavior: dragStartBehavior
     );
   }
```

Comment:
updateRenderObject isn't updated here. Please make sure to add a test that catches that error.

Reply:
Would it be sufficient to just set the drag start behavior in `updateRenderObject`? I'm not entirely sure what you mean by this also.